plugin/evm/vm_test.go

```diff
@@ -3153,24 +3153,61 @@ func TestCrossChainMessagestoVM(t *testing.T) {
 }
 
 func TestSignatureRequestsToVM(t *testing.T) {
-	_, vm, _, _ := GenesisVM(t, true, genesisJSONSubnetEVM, "", "")
+	_, vm, _, appSender := GenesisVM(t, true, genesisJSONSubnetEVM, "", "")
 
 	defer func() {
 		err := vm.Shutdown(context.Background())
 		require.NoError(t, err)
 	}()
 
-	// Generate a SignatureRequest for an unknown message
-	var signatureRequest message.Request = message.SignatureRequest{
-		MessageID: ids.GenerateTestID(),
-	}
-
-	requestBytes, err := message.Codec.Marshal(message.Version, &signatureRequest)
+	// Generate a new warp unsigned message and add to warp backend
+	warpMessage, err := avalancheWarp.NewUnsignedMessage(vm.ctx.ChainID, ids.GenerateTestID(), []byte{1, 2, 3})
 	require.NoError(t, err)
 
```

Comment:
nit: can slightly simplify with `require := require.New(t)`

Reply:
If we do that here, then when we use `t.Run(...)` require would have already taken in the outer `*testing.T` context.

I'd rather make the switch to use `require.New(t)` throughout this file in a separate PR